chalk-rust-ir/src/lib.rs

```diff
@@ -99,6 +103,15 @@ pub struct TraitDatum<I: Interner> {
 
     /// The id of each associated type defined in the trait.
     pub associated_ty_ids: Vec<AssocTypeId<I>>,
```

Comment:
The ids for the associated type members of the trait. The details of each type can be found by invoking the [`associated_type_data`](http://rust-lang.github.io/chalk/chalk_solve/trait.RustIrDatabase.html#tymethod.associated_ty_data) method.